cli/azd/pkg/exec/command_runner.go

```diff
@@ -252,11 +256,11 @@ func appendEnv(env []string) []string {
 	return nil
 }
 
-// newCmdTree creates a `CmdTree`, optionally using a shell appropriate for windows
+// NewCmdTree creates a `CmdTree`, optionally using a shell appropriate for windows
 // or POSIX environments.
 // An empty cmd parameter indicates "command list mode", which means that args are combined into a single command list,
 // joined with && operator.
-func newCmdTree(ctx context.Context, cmd string, args []string, useShell bool, interactive bool) (CmdTree, error) {
+func NewCmdTree(ctx context.Context, cmd string, args []string, useShell bool, interactive bool) (CmdTree, error) {
```

Comment:
Please revert this change `NewCmdTree` should be private and not accessible outside of the package.

Reply:
done